### ISSUE TITLE
Fill fileds from search

### ISSUE THREAD

**Issue by adocampo** [nextcloud/passman-webextension#282]
Hi guys! 
I wanted to writing this many months ago, and I never found the moment. But I think this passman behavior is so weird, it should be reported, so here I am.

At the moment, the search tool on the login form fields is pretty useless. It can search, but it cannot do anything useful, besides to show you have the credentials right there but you cannot use them. See image to figure what I'm trying to say.

![imagen](https://user-images.githubusercontent.com/2266612/46529627-71d1ec80-c897-11e8-9874-65fc77d3c661.png)

It would be awesome to let user pick the credentials and fill the username and password fields in this moment. Just like if passman had find the proper credentials. 
Case uses? Every time passman decided you don't have the proper credentials (sometimes, passman gets confused with subdomains, or with URLs generated on the fly, but you know which credentials are exactly).

### Steps to reproduce
1. Open a login form which passman doesn't know the credentials (or have some kind of problem with them)
2. Use the inline search tool
3. Try to click on the wanted credential

### Expected behaviour
Passman should fill username and password fields with the selected credentials obtained by the search result.

### Actual behaviour
Click does nothing.

### Configuration
**Operating system**:  Arch Linux

**Browser**: Firefox 62

**Extensions that might cause interference**: None

**Passman version**: 2.1.4

**Extension version**: 2.1.1

**Nextcloud version**: 13.0.6

**Comment by wjwieland** [comment]
I agree about the functionality of this popup.  I have attempted to edit entries in it, and they will work until one closes the current instance of chromium(or whatever browser), but those setting don't make it back to the server.....

**Comment by styx-tdo** [comment]
+1

**Comment by IBBoard** [comment]
I came to report a similar issue, but for the toolbar popup (it would be useful to copy from a search in there, rather than editing first). I hadn't thought of doing a copy from here!

From a quick bit of poking around, it looks like it should be fixed by #266? (Since this may be a duplicate of #251). Presumably we're waiting on a new release?

**Comment by Arc676** [comment]
Any update on this? This is still the case and it seems weird that this behavior is supposedly already fixed but the fixed version isn't released. That PR was merged over a year ago.

**Comment by binsky08** [comment]
This project has been completely rewritten (v3).
The old code is archived in the `legacy-v2` branch.
Please check whether the issue still exists in the current v3, and if so, please report it again.
